vcs/gogpm_test.go

```diff
@@ -14,7 +15,7 @@ func TestPresentPackageRepoSimpleGopath(t *testing.T) {
 		repo, _ := PackageForImportPath("github.com/fake/library")
 		repo.ctx = build.Context{Compiler: "gc", GOPATH: gopath}
 
-		expected := path.Join(gopath, "src", "/github.com/fake/library")
+		expected := path.Join(gopath, "src", "github.com", "fake", "library")
```

Comment:
it appears that `go/build` uses filepath internall

```
// joinPath calls ctxt.JoinPath (if not nil) or else filepath.Join.
    98	func (ctxt *Context) joinPath(elem ...string) string {
    99		if f := ctxt.JoinPath; f != nil {
   100			return f(elem...)
   101		}
   102		return filepath.Join(elem...)
   103	}
```

So i'll switch to consistently using filepath everywhere